README.rst

```diff
@@ -74,11 +74,14 @@ Settings
 
 ``FLICKR_TAG_IMAGE_SIZE`` - The size alias used if ``FLICKR_TAG_INCLUDE_DIMENSIONS`` is set to ``True``. Default is 'Medium 640'. See the `Flickr getSizes documentation`_ for the valid values. (Optional)
 
+``FLICKR_TAG_PLACE_HOLDER_PICT`` -  variable is mandatory in your config if you dont have ``FLICKR_API_KEY`` pointing to a local pict how serve as a placeholder
+
+``FLICKR_TAG_PLACE_HOLDER_LINK`` - link associated with placeholder pict Default to git repo of this plug-in
 
 Flickr Settings
 ---------------
 
-The following two settings are required. In order to set them up, you will need to set up a Flickr API key. You can do this by `creating an app on Flickr`_. If the blog is a personal blog, then apply for a non-commercial key. Once you've got your key and secret, add them to your `Pelican configuration`_.
+The following two settings are required. In order to set them up, you will need to set up a Flickr API key. You can do this by `creating an app on Flickr`_. If the blog is a personal blog, then apply for a non-commercial key. Once you've got your key and secret, add them apply `pelicanconf.py changes`_.
```

Comment:
the idea behind this is to exclude Flickr API key from pelicanconf.py file and then exclude the flickr_api.py file from version system like github.com